sklearn/metrics/scorer.py

```diff
@@ -529,9 +563,16 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 # Score function for probabilistic classification
 neg_log_loss_scorer = make_scorer(log_loss, greater_is_better=False,
                                   needs_proba=True)
+neg_brier_score_scorer = make_scorer(brier_score_loss,
+                                     greater_is_better=False,
+                                     needs_proba=True)
 brier_score_loss_scorer = make_scorer(brier_score_loss,
                                       greater_is_better=False,
                                       needs_proba=True)
+deprecation_msg = ('Scoring method brier_score_loss_scorer was renamed to '
```

Comment:
These aren't the right names

Reply:
sorry I approved too quickly, yes, these names are not correct. ``brier_score_loss_scorer`` should be ``brier_score_loss`` and ``neg_brier_score_loss_scorer`` should be ``neg_brier_score_loss``